workspaces/adventure-pack/css/style.css

```diff
@@ -21,3 +21,24 @@ body,
   font-weight: 400;
   font-style: normal;
 }
+
+.highlighted-code {
+  position: relative;
+}
+
+.copy-button {
+  position: absolute;
+  top: 5px;
+  right: 5px;
+  background: rgba(255, 255, 255, 0.7);
+  border: none;
+  border-radius: 4px;
+  padding: 4px 8px;
+  cursor: pointer;
+  opacity: 0;
```

Comment:
Random note, I'd be curious to understand the accessibility implications of 0 opacity, I believe some screen readers may treat it the same as visibility hidden: https://stackoverflow.com/a/43206381

We may also want to have opacity 1 when the button is focused, even if the container is not hovered.

But I think that's for a future investigation.